platform/posix/transport/src/openssl_posix.c

```diff
@@ -662,6 +662,9 @@ OpensslStatus_t Openssl_Disconnect( const NetworkContext_t * pNetworkContext )
 }
 /*-----------------------------------------------------------*/
 
+/* MISRA Rule 18.3 flags the following line for not using the const qualifier
```

Comment:
PR says 8.13, but comment says 18.3, which applies to using relational operators with pointers...